packages/css/src/plugins/parent-orphaned-pseudos.tsx

```diff
@@ -16,24 +16,29 @@ const prependNestingTypeToSelector = (selector: selectorParser.Node) => {
  *
  * E.g: `.class { &:hover {} }` will become `.class:hover {}`
  */
-export const parentOrphanedPseudos = plugin('parent-orphened-pseudos', () => {
-  return (root) => {
-    root.walkRules((rule) => {
-      const { selectors } = rule;
+export const parentOrphanedPseudos = (): Plugin => {
+  return {
+    postcssPlugin: 'parent-orphened-pseudos',
+    Once(root) {
+      root.walkRules((rule) => {
```

Comment:
`walkRules` can be replaced with a top-level `Rule` instead of `Once`

Reply:
I'm not sure if it's due to having a child call to walkPseudos, but I am unable to get the tests to pass after a few attempts.

Reply:
Okay, no problem! Thanks for trying :)